### DISCUSSION TITLE
Image getting api is hard to understand for beginner

### DISCUSSION THREAD

**Discussion by rafiul-hossain-riday** [Valexr/Slidy#100] [Q&A]
Is there any documentation which has something like this :
```svelte
let slides = [
    {id: 1, src: "/img/1.jpg"},
    {id: 2, src: "/img/2.jpg"},
    {id: 3, src: "/img/3.jpg"},
]
```
And then use the option? Documentation is supposed to be easy, also for svelte, right?
Got confused a bit, as I have less js knowledge but more svelte exp..

**Comment by EricRovell** [comment]
> Is there any documentation which has something like this :
> 
> ```svelte
> let slides = [
>     {id: 1, src: "/img/1.jpg"},
>     {id: 2, src: "/img/2.jpg"},
>     {id: 3, src: "/img/3.jpg"},
> ]
> ```
> 
> And then use the option? Documentation is supposed to be easy, also for svelte, right? Got confused a bit, as I have less js knowledge but more svelte exp..

Have you read the svelte specific [docs](https://github.com/Valexr/Slidy/tree/main/packages/svelte)?
Can you be more specific about your problem, please?
